client/client-lib/src/ln/db.rs

```diff
@@ -93,3 +97,22 @@ impl DatabaseKeyPrefixConst for LightningGatewayKey {
     type Key = Self;
     type Value = LightningGateway;
 }
+
+#[derive(Debug, Clone, Encodable, Decodable)]
+pub struct PaymentKey(pub sha256::Hash);
```

Comment:
This key should contain some sort of timestamp as the first field. Our databse is essentially an on-disk `BTreeMap`, so having the thing we (presumably?) want to sort by firs tin the key is very useful.